aws-redshift-cluster/src/test/java/software/amazon/redshift/cluster/TestUtils.java

```diff
@@ -42,6 +44,25 @@ public class TestUtils {
             .vpcSecurityGroups(Collections.emptyList())
             .build();
 
+    final static Cluster MULTIAZ_CLUSTER = Cluster.builder()
```

Comment:
can we create a common helper method in TestUtils to return a multiAZ cluster handy for unit tests ?

Reply:
I don't think we need a helper method for that. A static variable in utils class also has the same behavior.

Reply:
I see it's only for a single unit test, should be good